pyvista/utilities/cells.py

```diff
@@ -1,10 +1,43 @@
 """pyvista wrapping of vtkCellArray."""
+
+import sys
+from collections import deque
+from itertools import islice, count
+
 import numpy as np
 
 from pyvista import _vtk
 import pyvista
 
 
+def ncells_from_cells_py36(cells):
```

Comment:
Coverage boost again (to go with the no-cover pragma at the function call). Either this or add a test to make sure this still works...
```suggestion
def ncells_from_cells_py36(cells):  # pragma: no cover
```